crutest/src/cli.rs

```diff
@@ -223,8 +223,8 @@ enum CliCommand {
  * After verify, we truncate the data to 10 fields and return that so
  * the cli server can send it back to the client for display.
  */
-async fn cli_read(
-    guest: &Arc<Guest>,
+async fn cli_read<T: BlockIO + Send + Sync + 'static>(
```

Comment:
See my comment in the main PR conversation; this only actually requires `BlockIO` and can take a `&T` instead of an `&Arc<T>`

Reply:
Done